internal/pkg/cli/svc_init.go

```diff
@@ -170,6 +170,8 @@ func newInitSvcOpts(vars initSvcVars) (*initSvcOpts, error) {
 		mftReader:    ws,
 		dockerEngine: dockerengine.New(exec.NewCmd()),
 	}
+	// Override the app flag since we don't only allow running this command within a workspace.
```

Comment:
"Override the app flag since we don't only allow running this command within a workspace." 🤔

Reply:
I agree with @Lou1415926's suggestion. Or "Override the app flag because this command must be run within the app's workspace."?

Should we log a msg that the passed-in app is being overridden? Otherwise they won't realize that that's happening.

Reply:
sorry i misread the msg...corrected